spec/lib/spree/money_spec.rb

```diff
@@ -128,10 +118,14 @@
       expect(money.to_s).to eq("1.000.00 €")
     end
 
+    # rubocop:disable Layout/LineLength
     it "formats as HTML if asked (nicely) to" do
       money = Spree::Money.new(10)
       # The HTMLified version of the euro sign
-      expect(money.to_html).to eq("10.00&nbsp;&#x20AC;")
+      expect(money.to_html).to eq(
+        "<span&nbsp;class=\"money-whole\">10</span><span class=\"money-decimal-mark\">.</span><span class=\"money-decimal\">00</span> <span class=\"money-currency-symbol\">&#x20AC;</span>"
```

Comment:
I guess we should manually test places where money formatting is used, to make sure there's no regressions?

Reply:
I'll manually check but new `spans` with classes we don't use for styling should have no effect

Reply:
It works but it's weird that `money` returns this broken tag :thinking: 

![Screenshot from 2021-03-09 18-50-36](https://user-images.githubusercontent.com/762088/110515190-676f4900-8108-11eb-829a-fb99769d8ff0.png)

Reply:
Ugh... I don't like it.

Can we strip the tags on that output, so the previous specs for the output pass?

Reply:
fair enough.